app/models/site.rb

```diff
@@ -203,4 +203,16 @@ def self.clear_anon_cache!
     MessageBus.publish(SITE_JSON_CHANNEL, '')
   end
 
+  def self.show_welcome_topic_banner?(guardian)
+    return false unless guardian.is_admin?
+
+    first_admin_id = User.where(admin: true)
+      .human_users
+      .joins(:user_auth_tokens)
+      .order('user_auth_tokens.created_at')
+      .pluck_first(:id)
```

Comment:
I see we're using the same query in `lib/wizard.rb` as well, so maybe we should abstract this somewhere.